### ISSUE TITLE
woocommerce update

### ISSUE THREAD

**Issue by kurniaarifin27** [woocommerce/woocommerce#20385]
Whether updating woocommerce will delete the old file? 
I am worried about removing my previous, because I have added some custom functions

**Comment by kurniaarifin27** [comment]
ehmm this is what I'm worried about :(

is there no solution for woocommerce update without removing old files?

**Comment by mikejolley** [comment]
No. You should never edit plugin files - customisations should be done via the hooks/filters using custom plugin or a child theme you control.

**Comment by kurniaarifin27** [comment]
Oke very clear for me now,thanks for the explanation
